<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Port Commander</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/icons/icon/logo.png</normaloff>:/icons/icon/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background: rgb(40,40,40);
	color: white;
	border: 1px solid rgb(0,128,128);
}

QLabel
{
	border: none;
	background: none;
}

QTabWidget::pane {
    border: 1px solid rgb(0,128,128);
}

QFrame {
	border: none;
}

QTabWidget::tab-bar:top {
    top: 1px;
	border: none;
	background: none;
}

QTabBar::tab:top
{
	min-width: 100;
	min-height: 32;
}

QTabBar::tab:top:!selected {
	
	background: rgb(80,80,80);
 	border-bottom: 1px solid rgb(0,128,128);
}

QTabBar::tab:top:selected {
	background: rgb(40,40,40);
    border: 1px solid rgb(0,128,128);
	border-bottom-color: none;
}

QPushButton:checked
{
	background-color: rgba(0,128,128,64);
}

QCheckBox
{
	border: none;
}

QCheckBox::indicator {
     width: 16px;
     height: 16px;
}

QCheckBox::indicator:checked
{
	image: url(:/icons/icon/checked_checkbox.png);
}
QCheckBox::indicator:unchecked
{
	image: url(:/icons/icon/unchecked_checkbox.png);
}

QListWidget::indicator {
     width:16px;
     height: 16px;
}

QListWidget::indicator:checked
{
	image: url(:/icons/icon/checked_checkbox.png);
}
QListWidget::indicator:unchecked
{
	image: url(:/icons/icon/unchecked_checkbox.png);
}

QRadioButton::indicator {
     width:16px;
     height: 16px;
}

QRadioButton::indicator:checked
{
	image: url(:/icons/icon/checked_checkbox.png);
}
QRadioButton::indicator:unchecked
{
	image: url(:/icons/icon/unchecked_checkbox.png);
}
</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="left_bar">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <property name="spacing">
           <number>12</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="ports_title">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>PORTS</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListWidget" name="ports_list">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background: rgb(40,40,40);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="main_tab_widget"/>
    </item>
    <item row="0" column="2">
     <widget class="QFrame" name="right_bar">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="messages_frame">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <property name="spacing">
           <number>12</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="messages_title">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Messages</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="messages_file_path">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="messages_file_button">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res/res.qrc">
              <normaloff>:/icons/icon/file.png</normaloff>:/icons/icon/file.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QListWidget" name="messages_list">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background: rgb(40,40,40);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
